x-pack/docs/en/rest-api/security/has-privileges.asciidoc

```diff
@@ -29,11 +29,11 @@ privilege is assigned to the user.
 
 `index`::
 `names`::: (list) A list of indices.
-`allow_restricted_indices`::: (boolean) If `names` contains internal restricted
-that also have to be covered by the has-privilege check, then this has to be
-set to `true`. By default this is `false` because restricted indices should
-generaly not be "visible" to APIs. For most use cases it is safe to ignore
-this parameter.
+`allow_restricted_indices`::: (boolean) This needs to be set to `true` (default
```

Comment:
Is it worth adding one sentence in the end, something like "If restricted indices are explicitly included in the `names` list, privileges will be checked against them regardless of the value of `allow_restricted_indices`"  ?

Reply:
Yes, good idea, I've amended the description as you suggested.